src/libostree/ostree-repo-pull.c

```diff
@@ -5952,7 +5952,7 @@ find_remotes_cb (GObject *obj, GAsyncResult *async_result, gpointer user_data)
             {
               g_debug ("%s: Omitting remote ‘%s’ from results as none of its refs are new enough.",
                        G_STRFUNC, result->remote->name);
-              ostree_repo_finder_result_free (g_steal_pointer (&g_ptr_array_index (results, i)));
+              ostree_repo_finder_result_free (g_ptr_array_index (results, i));
```

Comment:
Coverity is confused here. [g_steal_pointer](https://docs.gtk.org/glib/func.steal_pointer.html) sets pointer it was passed to `NULL` and returns the pointed to value. So, the actual pointer is being returned. However, the address of the pointer is being set to `NULL`. That's actually important since the `results` pointer array is declared to free all the values in the array when the pointer array itself is freed. Without stealing the pointer, it will result in a double free.